packages/mock-block-dock/src/datastore/use-mock-datastore.ts

```diff
@@ -0,0 +1,618 @@
+  //           ) {
+  //             resolve({
+  //               errors: [
+  //                 {
+  //                   code: "NOT_FOUND",
+  //                   message: `Could not find linked aggregation with aggregationId '${data.aggregationId}'`,
+  //                 },
+  //               ],
+  //             });
+  //             return currentLinkedAggregations;
+  //           }
+  //           return currentLinkedAggregations.map((linkedAggregation) => {
+  //             if (linkedAggregation.aggregationId === data.aggregationId) {
+  //               const newLinkedAggregation = {
+  //                 ...linkedAggregation,
+  //                 operation: data.operation,
+  //               };
+  //               resolve({ data: newLinkedAggregation });
+  //               return newLinkedAggregation;
+  //             }
+  //             return linkedAggregation;
+  //           });
+  //         });
+  //       });
+  //     },
+  //     [setLinkedAggregations, readonly],
+  //   );
+  //
+  // const deleteLinkedAggregation: EmbedderGraphMessageCallbacks["deleteLinkedAggregation"] =
+  //   useCallback(
+  //     async ({ data }) => {
+  //       if (readonly) {
+  //         return readonlyErrorReturn;
+  //       }
+  //
+  //       if (!data) {
+  //         return {
+  //           errors: [
+  //             {
+  //               code: "INVALID_INPUT",
+  //               message: "deleteLinkedAggregation requires 'data' input",
+  //             },
+  //           ],
+  //         };
+  //       }
+  //       return new Promise((resolve) => {
+  //         setLinkedAggregations((currentLinkedAggregations) => {
+  //           if (
+  //             !currentLinkedAggregations.find(
+  //               ({ aggregationId }) => aggregationId === data.aggregationId,
+  //             )
+  //           ) {
+  //             resolve({
+  //               errors: [
+  //                 {
+  //                   code: "NOT_FOUND",
+  //                   message: `Could not find link with aggregationId '${data.aggregationId}'`,
+  //                 },
+  //               ],
+  //             });
+  //             return currentLinkedAggregations;
+  //           }
+  //           return currentLinkedAggregations.filter((link) => {
+  //             if (link.aggregationId === data.aggregationId) {
+  //               resolve({ data: true });
+  //               return false;
+  //             }
+  //             return true;
+  //           });
+  //         });
+  //       });
+  //     },
+  //     [setLinkedAggregations, readonly],
+  //   );
+
+  const uploadFile: EmbedderGraphMessageCallbacks["uploadFile"] = useCallback(
+    async ({ data }) => {
+      return {
+        errors: [
+          {
+            code: "NOT_IMPLEMENTED",
+            message: `Uploading files is not currently supported`,
+          },
+        ],
+      };
+
+      /** @todo - create the file entity-type and re-enable file uploading */
```

Comment:
This seemed out of scope for this PR, I think it'd have to be a Block Protocol managed type and that's a whole can of worms